### DISCUSSION TITLE
Is it possible to change canvas/waveform?

### DISCUSSION THREAD

**Discussion by Yi-Zhou-Zhou** [samhirtarif/react-audio-recorder#92] [Q&A]
**Describe the bug**

Hi,

Is there any way i can change/style the canvas to show a waveform similar to the following image?

**Screenshots**
![image](https://github.com/samhirtarif/react-audio-recorder/assets/65474467/53d89dc4-a137-48cf-95d9-34e547902fae)

**Comment by samhirtarif** [comment]
@Yi-Zhou-Zhou - You can change the background color of the entire component. But as of now, the bar colors for the visualizer are not modifiable. I can add that feature if you need it :)

Here is an example of how to modify the background color
https://stackblitz.com/edit/react-ts-uboso9?file=style.css,App.tsx,index.tsx

**Reply by Yi-Zhou-Zhou** [reply]
I've just found [your other library](https://github.com/samhirtarif/react-audio-visualize) which did the trick for me.

Thanks!
